recipes/waterfallcharts/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "waterfallcharts" %}
+{% set version = "3.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/waterfallcharts-{{ version }}.tar.gz
+  sha256: 01deb3464968bf6481f3b8b2cb09f2fb1011f1865fbb608e0520d7957bedd7b1
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python  >=3.5
+    # the following dependencies are not correctly specified upstream
+    - numpy
+    - pandas
+    - matplotlib
```

Comment:
```suggestion
    - matplotlib-base
```

Reply:
Makes sense, thanks. Looks like everything still passes with this change - good to merge?